framework/pom.xml

```diff
@@ -39,6 +39,17 @@
             <artifactId>commons-io</artifactId>
         </dependency>
 
+        <!-- FTP -->
+        <dependency>
```

Comment:
I don't think this should go into the framework.
Is it possible to move this code and all related code to the new ftp module only?